tools/pipelines/test-perf-benchmarks.yml

```diff
@@ -118,13 +118,18 @@ stages:
           inputs:
             targetType: 'inline'
             script: |
-              echo "Unpack test files for ${{ testPackage }}"
-              TAR_FILE=${{ replace(replace(replace(replace(testPackage, '@fluidframework/', '' ), '@fluid-internal/', '' ),'@fluid-', '' ), '/', '-') }}
-              mkdir -p ${{ variables.testWorkspace }}/node_modules/${{ testPackage }}/src/test
-              mkdir -p ${{ variables.testWorkspace }}/node_modules/${{ testPackage }}/dist/test
-              tar -xvf $(testFilesPath)/$TAR_FILE.test-files.tar -C $(testFilesPath)
-              mv $(testFilesPath)/dist/test/* ${{ variables.testWorkspace }}/node_modules/${{ testPackage }}/dist/test
-              mv $(testFilesPath)/src/test/* ${{ variables.testWorkspace }}/node_modules/${{ testPackage }}/src/test
+              TAR_FILENAME=${{ replace(replace(replace(replace(testPackage, '@fluidframework/', '' ), '@fluid-internal/', '' ),'@fluid-', '' ), '/', '-') }}
```

Comment:
This... frightens me... I wonder where else in the pipeline we have hardcoded expectations about scopes and package names?

Reply:
In both cases that functionality _should_ be extracted into something testable and runnable outside CI. At least we're not running out of work? :)